app/src/debug/java/org/schabi/newpipe/DebugApp.kt

```diff
@@ -0,0 +1,58 @@
+package org.schabi.newpipe
+
+import android.content.Context
+import androidx.multidex.MultiDex
+import androidx.preference.PreferenceManager
+import com.facebook.stetho.Stetho
+import com.facebook.stetho.okhttp3.StethoInterceptor
+import leakcanary.AppWatcher
+import leakcanary.LeakCanary
+import okhttp3.OkHttpClient
+import org.schabi.newpipe.extractor.downloader.Downloader
+
+class DebugApp : App() {
+    override fun attachBaseContext(base: Context) {
+        super.attachBaseContext(base)
+        MultiDex.install(this)
+    }
+
+    override fun onCreate() {
+        super.onCreate()
+        initStetho()
+
+        AppWatcher.config = AppWatcher.config.copy(watchDurationMillis = 10000)
```

Comment:
Please (re-)add the comment about the delay. The new naming is less understandable.